tests/setup-validators.sh

```diff
@@ -73,7 +73,7 @@ docker cp ./genesis.json $VALIDATOR_CONTAINER_BASE_NAME$i:/validator/config/gene
 
 VALIDATOR_CONTAINER_IP=$(docker inspect -f '{{range.NetworkSettings.Networks}}{{.IPAddress}}{{end}}' $VALIDATOR_CONTAINER_BASE_NAME$i)
 
-docker exec $VALIDATOR_CONTAINER_BASE_NAME$i $BIN gentx $ARGS --moniker validator$i --chain-id=$CHAIN_ID --ip $VALIDATOR_CONTAINER_IP validator 500000000stake
+docker exec $VALIDATOR_CONTAINER_BASE_NAME$i $BIN gentx $ARGS --moniker validator$i --chain-id=$CHAIN_ID --ip $VALIDATOR_CONTAINER_IP validator 1000000stake
```

Comment:
easier to use 1 mil stake for all validators to start, just so its easier to read values during testing